specs/glossary.md

```diff
@@ -591,7 +591,7 @@ In the current implementation, this is the L1 block number at which the output o
 
 [safe-l2-block]: glossary.md#safe-l2-block
 
-A safe L2 block is an L2 block can be derived entirely from L1 by a [rollup node][rollup-node]. This can vary between
+A safe L2 block is an L2 block that can be derived entirely from L1 by a [rollup node][rollup-node]. This can vary between
```

Comment:
fix is good, but line is now too long & fails markdown lint.

Reply:
Fixed. See e329332.